src/BlockGeoMaker.js

```
@@ -102,13 +102,22 @@ export default class BlockGeoMaker extends AsyncFactory {
+				face["normal"] = this.#applyEulerRotation(face["normal"], rotation, [0, 0, 0]);
 				face["vertices"].forEach(vertex => {
 					vertex["pos"] = this.#applyEulerRotation(vertex["pos"], rotation, [8, 8, 8]); // (8, 8, 8) is the block center and the pivot for all block-wide rotations
 				});
 			});
 			centerOfMass = this.#applyEulerRotation(centerOfMass, rotation, [8, 8, 8]);
 		}
 		faces = this.#scaleFaces(faces, centerOfMass);
+		faces.forEach(face => {
+			if(face["fullbright"]) {
+				face["normal"] = [0, 1, 0];
+				delete face["fullbright"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)